jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -60,7 +63,7 @@ public synchronized static boolean isCRIUSupportEnabled() {
 			init();
```

Comment:
we should ifdef the init path and just return false here if criu is not enabled

Reply:
fixed.